.circleci/config.yml

```diff
@@ -51,7 +50,7 @@ make_env: &make_env
   name: Make workshop environment
   command: |
     conda env create -n astropy-workshop --file 00-Install_and_Setup/environment.yml
-    conda activate astropy-workshop
+    source activate astropy-workshop
```

Comment:
Wait... I thought `source activate` is a very old command that should not be used anymore, no?

Reply:
Going based on this: https://stackoverflow.com/questions/58849994/running-tests-at-circleci-in-conda-environment

It seems I can either do: 
```
conda init bash
source ~/.bashrc
conda activate astropy-workshop
```
or
``
source activate astropy-workshop
```
🤷‍♂

Reply:
Is the conda version on the CircleCI docker image too old or something?

Reply:
I don't think there is one, but link me to it if I'm wrong!